operator/src/main/resources/scripts/startServer.sh

```diff
@@ -26,6 +26,8 @@ for varname in domain_uid \
                domain_home \
                log_home \
                nodemgr_home \
+               service_name \
+               admion_hostname \
```

Comment:
Typo - admion --> admin

Reply:
Good eye. It is in a debug/info statement so testing did not catch it. fixed.